src/hotspot/share/runtime/vmOperations.hpp

```diff
@@ -143,12 +143,14 @@ class VM_PrintThreads: public VM_Operation {
   outputStream* _out;
   bool _print_concurrent_locks;
   bool _print_extended_info;
+  bool _print_jni;
  public:
   VM_PrintThreads()
-    : _out(tty), _print_concurrent_locks(PrintConcurrentLocks), _print_extended_info(false)
+    : _out(tty), _print_concurrent_locks(PrintConcurrentLocks), _print_extended_info(false), _print_jni(false)
   {}
```

Comment:
This no-arg constructor is unused now.

Reply:
the no-arg  constructor is used by JVM_DumpAllStacks now.